examples/files.json

```diff
@@ -426,7 +426,8 @@
 		"webgpu_video_panorama",
 		"webgpu_volume_cloud",
 		"webgpu_volume_perlin",
-		"webgpu_water"
+		"webgpu_water",
+		"webgpu_material_arrays"
```

Comment:
Do you mind renaming the example to `webgpu_materials_arrays` and place it below `webgpu_materials` in the list?

Reply:
Done!